slack_bolt/app/app.py

```diff
@@ -912,6 +913,7 @@ def function(
         matchers: Optional[Sequence[Callable[..., bool]]] = None,
         middleware: Optional[Sequence[Union[Callable, Middleware]]] = None,
         auto_acknowledge: bool = True,
+        acknowledgement_timeout: int = 3,
```

Comment:
Should we name this `ack_timout` rather then `acknowledgement_timeout`?

Reply:
thought: `ack_timeout` makes sense to me and matches the `ack` parameter well! 📚 ✨